docs/.vitepress/components/Glossary.vue

```diff
@@ -0,0 +1,43 @@
+<!-- Taken and modified from https://gitlab.com/eeriksp/vuepress-plugin-glossary -->
```

Comment:
Why exactly did we need to modify it? What about the licence?

Reply:
Removed props, which I guess we could restore and then pass the data still. Simplified